lib/pixify.js

```diff
@@ -118,8 +128,8 @@ module.exports = function(options, callback) {
                 }))
                 .pipe(gulpif(options.compress, uglify()))
                 .pipe(header(license, {
-                    date: (new Date()).toString(),
-                    name: packageInfo.name,
+                    date: (new Date()).toUTCString(),
+                    name: options.name,
```

Comment:
Alright, might then just expose options to the license template. I assumed it wasn't used because the template in lib/license.js doesn't make use of it.